src/token.rs

```diff
@@ -484,6 +485,7 @@ macro_rules! Token {
     (,)        => { $crate::token::Comma };
     (/)        => { $crate::token::Div };
     (/=)       => { $crate::token::DivEq };
+//  ($)        => { $crate::token::Dollar }; //TODO: Enable when upgrading to Rust >= 1.16
```

Comment:
I would prefer to remove these commented lines and just file a GitHub issue to follow up when we eventually bump the required compiler version. Having an issue makes it easier to label and prioritize and track the work.